Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -197,10 +197,18 @@ public class SchemeGenerator {
         let launchVariables = scheme.run.flatMap { $0.environmentVariables.isEmpty ? nil : $0.environmentVariables }
         let profileVariables = scheme.profile.flatMap { $0.environmentVariables.isEmpty ? nil : $0.environmentVariables }
 
+        let testPlans = scheme.test?.testPlans.map {
+            testPlan in XCScheme.TestPlanReference(
+                reference: "container:\(testPlan.path)",
```

Comment:
Small thing but:
The code should stick to one type of string operation. on some places it uses concatenation and then interpolation. At some places we use `a + b` on this here `"xy\()"`

Here: 
```swift
"container:\(testPlan.path)"
```

`Tests/XcodeGenKitTests/SchemeGeneratorTests.swift`
```swift
 let testPlanPath = fixturePath.string + "/TestProject/App_iOS/App_iOS.xctestplan"
```

Reply:
Changed in 642ba14